app/components/primer/experimental/action_bar.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Primer
+  module Experimental
+    # Add a general description of component here
+    # Add additional usage considerations or best practices that may aid the user to use the component correctly.
+    # @accessibility Add any accessibility considerations
+    class ActionBar < Primer::Component
+      status :experimental
+
+      SIZE_DEFAULT = :medium
+      SIZE_MAPPINGS = {
+        SIZE_DEFAULT => nil,
+        :small => "ActionBar--small",
+        :large => "ActionBar--large"
+      }.freeze
+      SIZE_OPTIONS = SIZE_MAPPINGS.keys.freeze
```

Comment:
Added this back to make sure there is enough spacing between items so that the touch targets don't overlap.

![Screen Shot 2022-08-31 at 14 08 40](https://user-images.githubusercontent.com/378023/187599121-f92c0289-66c5-4166-89e7-9b436a861460.png)